torchvision/transforms/v2/_geometry.py

```diff
@@ -590,9 +590,16 @@ class RandomRotation(Transform):
         expand (bool, optional): Optional expansion flag.
             If true, expands the output to make it large enough to hold the entire rotated image.
             If false or omitted, make the output image the same size as the input image.
-            Note that the expand flag assumes rotation around the center and no translation.
+            Note that the expand flag assumes rotation around the center (see note below) and no translation.
         center (sequence, optional): Optional center of rotation, (x, y). Origin is the upper left corner.
             Default is the center of the image.
+
+            ..note::
+
+              In theory, setting ``center`` has no effect if ``expand=True``, since the image center will become the
+              center of rotation. In practice however, due to numerical precision, this can lead to off-by-one
+              differences of the resulting image size compared to using the image center in the first place. Thus, when
+              setting ``expand=True``, it's best to leave ``center=None`` (default).
```

Comment:
Indentation

```suggestion
            .. note::

                In theory, setting ``center`` has no effect if ``expand=True``, since the image center will become the
                center of rotation. In practice however, due to numerical precision, this can lead to off-by-one
                differences of the resulting image size compared to using the image center in the first place. Thus, when
                setting ``expand=True``, it's best to leave ``center=None`` (default).
```